www/layouts/educator/section.html

```diff
@@ -51,7 +51,7 @@
             data-toggle="collapse"
             data-target=".navbar-collapse.show"
             href="#share-adaptions"
-            >Share Adaptations</a
+            >Share Adaptions</a
```

Comment:
@pdpinch is this an industry term or something we're using? I think `adaptation` is definitely a more commonly used word - I thought `adaption` was not a word until I looked it up and got 'synonym for adaptation'

Reply:
Yeah, that needs to be changed everywhere. It was one of the checkboxes in the issue description, but it's not checked yet. 

> typo "Adaptation" instead of "Adaption" throughout the page, including file names and ids

Reply:
I think @MAbdurrehman12 changed them already, I was just raising a question about the word - but that may be neither here nor there 🙂

Reply:
Yeah, I changed it everywhere

Reply:
Is there something to be discussed in this conversation or should we resolve it?

Reply:
@pdpinch I guess @alicewriteswrongs is referring to the commonness, as both, `adaptation` and `adaption` have the same meaning. However, `adaptation` is more commonly used.

Reply:
Sorry, I caused some confusion here. "Adaptation" is the preferred term in this context.

Reply:
Sure
No worries